tests/requests/test_accept_header.py

```diff
@@ -0,0 +1,101 @@
+from unittest import TestCase
+
+from django.http import HttpRequest
+from django.http.request import MediaType
+
+
+class MediaTypeTests(TestCase):
+    def test_empty(self):
+        for empty_media_type in (None, ''):
+            with self.subTest(media_type=empty_media_type):
+                media_type = MediaType(empty_media_type)
+                self.assertIs(media_type.is_all_types, False)
+                self.assertEqual(str(media_type), '')
+                self.assertEqual(repr(media_type), '<MediaType: >')
```

Comment:
This doesn't look reasonable but an empty `Accept` header is valid per [RFC7231#section-5.3.2](https://tools.ietf.org/html/rfc7231#section-5.3.2).